core/src/test/scala/ConfigSpec.scala

```diff
@@ -0,0 +1,10 @@
+import codesearch.core.config.DatabaseConfig
+import org.scalatest.{FlatSpec, Matchers}
+
+class ConfigSpec extends FlatSpec with Matchers {
+  val config = DatabaseConfig.load()
+
+  "Config" should "be not null" in {
+    config.right.get should not be null
```

Comment:
Right projection of either does not return null, it throws error if empty. Try something like 
```
config shouldBe Right(
  Config(
    DatabaseConfig(
    ....
    )
  )
)
```